src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Performance/ProjectSystem/BackgroundCodeGenerationBenchmark.cs

```diff
@@ -31,7 +31,7 @@ public void Cleanup()
         private DefaultProjectSnapshotManager SnapshotManager { get; set; }
 
         [Benchmark(Description = "Generates the code for 100 files", OperationsPerInvoke = 100)]
-        public async Task BackgroundCodeGeneration_Generate100Files()
+        public async Task BackgroundCodeGeneration_Generate100FilesAsync()
```

Comment:
I wouldn't imagine this would be needed for a benchmark project

Reply:
➡️ I certainly wouldn't complain about updating [RelaxTestNamingSuppressor.cs](https://github.com/dotnet/roslyn-analyzers/blob/0ed91b06dd96f30b0ac51ac13df9cc6508892f9f/src/Roslyn.Diagnostics.Analyzers/Core/RelaxTestNamingSuppressor.cs) to recognize `BenchmarkAttribute`, but currently it doesn't. It doesn't seem like a common case though so maybe lower priority?

Reply:
Super low priority, was just an observation